src/Sandbox.jl

```diff
@@ -89,16 +89,15 @@ function select_executor(verbose::Bool)
     error("Could not find any available executors for $(triplet(HostPlatform()))!")
 end
 
-_preferred_executor = nothing
+const _preferred_executor = Ref{Union{Nothing, (x->Type{x}).(all_executors)...}}(nothing)
```

Comment:
Not sure this helps anything in practice, `all_executors` has already three elements, with more possibly to come.